packages/browser/webpack.common.js

```diff
@@ -20,6 +20,11 @@ module.exports = {
   },
   resolve: {
     extensions: [".tsx", ".ts", ".js"],
+    fallback: {
```

Comment:
So I see the references in `package.json`, and this here, but I didn't notice any usage of these libraries in this PR (but perhaps I missed it due to the noise of all the ESLint updates). But basically, why are these (and `package.json`) change required as part of this PR...?

Reply:
That's to replace Webpack's polyfills for NodeJS builtins. Our code don't depend on these directly, but transitively through `node-jose` and `jsonwebtoken`. These polyfills are needed to generate a browser bundle.

Reply:
Might be a good idea to leave a comment next to our usage of node-jose and here that we can remove these packages as well when we remove it.